# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/workgroup.comp
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/workgroup.comp b/third_party/rust/spirv-cross-internal/tests/shaders/workgroup.comp
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/workgroup.comp
+++ /dev/null
@@ -1,9 +0,0 @@
-#version 450
-
-layout(local_size_x_id = 5, local_size_y_id = 10, local_size_z_id = 15) in;
-
-void main()
-{
-    // See https://github.com/KhronosGroup/glslang/issues/557
-    gl_WorkGroupSize;
-}